ghost/core/core/server/api/endpoints/utils/serializers/output/mappers/emails.js

```diff
@@ -7,13 +6,13 @@ module.exports = (model, frame) => {
 
     // Ensure we're not outputting unwanted replacement strings when viewing email contents
     // TODO: extract this to a utility, it's duplicated in the email-preview API controller
-    const replacements = mega.postEmailSerializer.parseReplacements(jsonModel);
-    replacements.forEach((replacement) => {
-        jsonModel[replacement.format] = jsonModel[replacement.format].replace(
-            replacement.regexp,
-            replacement.fallback || ''
-        );
-    });
+    // const replacements = mega.postEmailSerializer.parseReplacements(jsonModel);
```

Comment:
Need to check why this is needed

Reply:
This is used in older emails that still have html bodies. Added a new implementation that is covered by tests